app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_comment, only: %i[destroy]
+
+  def create
+    @comment = @commentable.comments.build(comment_params)
+    @comment.user = current_user
+    if @comment.save
+      redirect_to @commentable, notice: t('controllers.common.notice_create', name: Comment.model_name.human)
+    else
+      redirect_to @commentable, alert: t('controllers.common.needs_input', name: Comment.model_name.human)
```

Comment:
```suggestion
      redirect_to @commentable, alert: @comment.errors.full_messages
```

みたいな形でバリデーションエラーのエラーメッセージを再利用する方がベターです。